cloudca/resource_cloudca_vpn.go

```diff
@@ -0,0 +1,147 @@
+package cloudca
+
+import (
+	"fmt"
+
+	"github.com/cloud-ca/go-cloudca"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceCloudcaVpn() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudcaVpnCreate,
+		Read:   resourceCloudcaVpnRead,
+		Delete: resourceCloudcaVpnDelete,
+
```

Comment:
The `Importer` is missing here, specially that you've already added the [import section](https://github.com/cloud-ca/terraform-provider-cloudca/blob/vpn/doc/vpn.md#import) in the doc.

You can add it like [this](https://github.com/cloud-ca/terraform-provider-cloudca/blob/master/cloudca/resource_cloudca_instance.go#L20-L22):

```
Importer: &schema.ResourceImporter{
	State: schema.ImportStatePassthrough,
},
```

Reply:
Ok, I will add this...

Reply:
+1

Reply:
+1